cfme/tests/v2v/test_post_migrations.py

```diff
@@ -102,3 +118,7 @@ def test_migration_post_attribute(appliance, provider, mapping_data_vm_obj_mini,
     soft_assert(source_core == target_core)
     # Test7: memory on source and target vm
     soft_assert(source_memory == target_memory)
+    # Test8: check snapshot should be removed after migration
+    expected_count = 1 if provider.one_of(RHEVMProvider) else 0
+    ui_snap_count = target_view.entities.summary("Properties").get_text_of("Snapshots")
+    soft_assert(ui_snap_count == expected_count)
```

Comment:
soft assert is not needed here as this is the last assert statement

Reply:
please use assert instead.  soft_assert should be used only in case when there can be several asserts in one test and we would like to see results of all failed asserts.

one more thing, if you put assert, please add description to it explaining what's wrong.